### PR TITLE
Remove unused resources

### PR DESCRIPTION
**Pull request by fluiddot**
This PR removes resources that are no longer referenced. Most likely derived by forgetting to remove them when deleting test cases.

### CODE REVIEW

File: test-cases/resources/cancel-upload.png
Comment (dcalhoun) [inline test-cases/resources/cancel-upload.png:1]: I believe we need to retain this. It appears to still be referenced. 

https://github.com/wordpress-mobile/test-cases/blob/70a701e7b831fc59a49522e6dabc9b597308c55e/test-cases/gutenberg/image.md?plain=1#L33

**Review by dcalhoun — APPROVED** [review]
Aside from the one accidental removal, this looks good to me. Thanks!

File: test-cases/resources/cancel-upload.png
Comment (fluiddot) [reply test-cases/resources/cancel-upload.png:1]: Yes, but we point to the `JPG` version. We can either remove the `PNG` file or use that version and remove `JPG.

Comment (dcalhoun) [reply test-cases/resources/cancel-upload.png:1]: Ah, OK. I overlooked the file ending. 👍🏻

Comment (fluiddot) [reply test-cases/resources/cancel-upload.png:1]: No worries 🙂, the same happened to me when I remove it.

**Pull request merged**